x-pack/plugins/security_solution/public/attack_discovery/links.test.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ASSISTANT_FEATURE_ID } from '../../common/constants';
+import { SERVER_APP_ID } from '../../common';
+import { links } from './links';
+
+describe('links', () => {
+  it('for serverless, it specifies capabilities as an OR condition, via a nested array', () => {
```

Comment:
The nested array is for AND conditions, both capabilities are required, isn't it?
```suggestion
  it('for serverless, it specifies capabilities as an AND condition, via a nested array', () => {
```
I am not sure about the need for this test though. This config is the source of truth, and any change here should trigger the gen-ai team review, maybe a comment explaining why capabilities is defined with a double array would be enough. But as you prefer, it's not going to hurt either.